android/assets/jsons/translations/German.properties

```diff
@@ -1333,7 +1310,9 @@ Tithe = Zehnt
 World Church = Weltkirche
 
 Defender of the Faith = Verteidiger des Glaubens
-Enhancer = Verbesserung
+
+Enhancer = Verstärker
```

Comment:
Wie bereits in [Kommentar](https://github.com/yairm210/Unciv/pull/5020#issuecomment-908261083) beschrieben, ist es eher eine "Verbesserung" und kein "Verstärker"